csv2sql.drush.inc

```diff
@@ -45,46 +46,53 @@ function drush_csv2sql($csv_path) {
     return;
   }
 
-  if (!file_exists($csv_path)) {
+  // Check if the command has a file direct path.
+  $files = $csv_path ? array($csv_path) : array();
+  // Check if the command has a directory path to scan.
+  if ($dir = drush_get_option('dir')) {
+    $files = csv2sql_scan_for_csv_files($dir);
+  }
+
+  if (!file_exists($csv_path) && empty($files)) {
```

Comment:
Only if you give a file and a directory at the same time, But it doesn't matter, I think yes I should change it to receive either a file or a directory anyway like you suggested.